test/tests/api/v2_0/skupack_tests.py

```diff
@@ -244,13 +242,29 @@ def post_skupack(self):
                     Api().profiles_get_metadata_by_name('useless.json', scope=sku_id)
                     assert_equal(200, self.__client.last_response.status)
                     assert_equal(1, len(loads(self.__client.last_response.data)))
-
+                    
                     """Test DELETE:api/2.0/skus/:identifier/pack"""
+                    Api().skus_get()
+                    skus = loads(self.__client.last_response.data)
+                    LOG.info("Printing Sku from get skus before a delete sku pack")
+                    LOG.info(skus)
                     Api().skus_id_delete_pack(self.__packFolderId)
                     Api().skus_id_delete(self.__packFolderId)
                     result = self.__client.last_response
                     assert_equal(200, result.status, message=result.reason)
-
+                    data = loads(result.data)
+                    LOG.info("ID of the deleted sku pack is:   "+ data.get('id'))
+                    Api().skus_get()
+                    skus = loads(self.__client.last_response.data)
+                    LOG.info("List of Skus after delete sku pack")
+                    LOG.info(skus)
+                    assert_equal(0, len(skus))
+                    for n in skus:
```

Comment:
Seems this for loop is useless due to the above assertion, as the skus length will always be zero here.